packages/tools/client-debugger/client-debugger-view/src/RenderClientDebugger.tsx

```diff
@@ -22,13 +24,28 @@ import { FluidClientDebuggers } from "./Debugger";
  */
 export async function renderClientDebuggerView(targetElement: HTMLElement): Promise<void> {
 	const debuggerElement = document.createElement("debugger");
-	targetElement.append(debuggerElement);
 
 	return new Promise<void>((resolve, reject) => {
 		try {
-			ReactDOM.render(React.createElement(FluidClientDebuggers), debuggerElement, resolve);
+			ReactDOM.render(<RootView />, debuggerElement, () => {
+				targetElement.append(debuggerElement);
```

Comment:
Is this necessary? ReactDOM.render should already do this.

Reply:
React will append the <RootView /> DOM tree to `debuggerElement`, but we still need `debuggerElement` appended to the `targetElement` passed to the function, right? I just brought this line from outside for consistency with how RootView.tsx in the chrome extension does it. We could maybe get rid of the `debuggerElement` altogether and just render <RootView /> straight onto the `targetElement`?

Reply:
I ended up creating that element outside the function call and passing that instead of the previous arg... which was `document.body`, so rendering the debugger made the app itself diseappear 😄 .